### ISSUE TITLE
Make all paths absolute

### ISSUE THREAD

**Issue by bookdude13** [ProtonLights/cli#21]
Low priority, but throwing it out there for future reference.

Right now a lot of file paths are built around the assumption that the user is running proton in a project directory. While this should be the case, having code be directory-dependent messes with our tests (can't run in parallel since they change the current directory). It would be nice in the distant future to have only absolute paths used.
